libs/components/layout/src/lib/modules/box/box-header.component.scss

```diff
@@ -0,0 +1,18 @@
+.sky-box .sky-box-header-content sky-box-header {
```

Comment:
Can this just use `:host`? I don't think we need to add the other qualifiers to the selector (`.sky-box`, `.sky-box-header-content`, etc.) as it will just generate excessive CSS for the nested `.sky-box-header` element.